src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "xebia.cakemix.ask-timeout"
```

Comment:
Also, https://groups.google.com/forum/#!searchin/scala-user/Package$20Naming$20Conventions/scala-user/3xWcfGYBwec/_3FDMFnjPdkJ .

Given that you can do http://stackoverflow.com/questions/28238520/python-like-package-name-aliasing-in-scala I think I like the shorter version. Having 'xebia' in there does have some vanity value I suppose, OTOH there's beauty in having this stand 'on its own'.

Xebia is still in the 'organization' of the package and the repo is under 'Xebia' in github, I'm leaning towards just 'cakemix' as package.